app/src/main/java/com/gpayinterns/chat/ChatContentProvider.java

```diff
@@ -1,134 +0,0 @@
-package com.gpayinterns.chat;
-
-import android.content.ContentUris;
-import android.content.ContentValues;
-import android.content.UriMatcher;
-import android.database.Cursor;
-import android.database.sqlite.SQLiteDatabase;
-import android.net.Uri;
-
-public class ChatContentProvider extends android.content.ContentProvider
-{
-    private OpenHelper mDbOpenHelper;
-
-    public static UriMatcher sURIMatcher = new UriMatcher(UriMatcher.NO_MATCH);
-
-    public static int MESSAGES = 0;
-    public static int USERS = 1;
-    public static int CHAT = 2;
-    static
-    {
-        sURIMatcher.addURI(ChatProviderContract.AUTHORITY, ChatProviderContract.Messages.PATH, MESSAGES);
-        sURIMatcher.addURI(ChatProviderContract.AUTHORITY, ChatProviderContract.Users.PATH,USERS);
-        sURIMatcher.addURI(ChatProviderContract.AUTHORITY, ChatProviderContract.Chat.PATH,CHAT);
-
-    }
```

Comment:
All database related files have been removed as my application currently doesn't support client side caching. It will be added later on if reqd.